Tasks/Common/ServiceFabricHelpers/Connect-ServiceFabricClusterFromServiceEndpoint.ps1

```diff
@@ -199,6 +215,7 @@ function Connect-ServiceFabricClusterFromServiceEndpoint
         # Reset the scope of the ClusterConnection variable that gets set by the call to Connect-ServiceFabricCluster so that it is available outside the scope of this module
         Set-Variable -Name ClusterConnection -Value $Private:ClusterConnection -Scope Global
 
+         return $RefreshToken
```

Comment:
This is not ideal. RefreshToken can only be understood by this script file. Caller can't use it - it will just pass it back. Can we use a script variable - since this is a module it should retain variable value across function calls

Reply:
Pass RefreshToken by reference is another option. Let me do that